plugins/ssr/ssrelim.ml

```diff
@@ -28,7 +28,7 @@ module RelDecl = Context.Rel.Declaration
 
 (** The "case" and "elim" tactic *)
 
-let apply_type x xs = Proofview.V82.of_tactic (Tactics.apply_type ~typecheck:false x xs)
+let apply_type x xs = Proofview.V82.of_tactic (Tactics.apply_type ~typecheck:true x xs)
```

Comment:
Why redefine this here? Isn't it the same as `Ssrcommon.apply_type`?

Reply:
I guess so, and I'm afraid I'm not the one that defined apply_type in every file ;-)